registry/quay.io/biocontainers/mrbayes/container.yml

```diff
@@ -0,0 +1,14 @@
+docker: quay.io/biocontainers/mrbayes
```

Comment:
This recipe file isn't being detected because `container.yml` should be renamed to `container.yaml`, see error in test here! https://github.com/singularityhub/singularity-hpc/pull/417/checks?check_run_id=2987039644

Reply:
oh yep sorry!!